src/test/ui/numeric/uppercase-base-prefix-invalid-no-fix.rs

```diff
@@ -0,0 +1,31 @@
+fn main() {
```

Comment:
Can you add a brief comment about the purpose of the test, something like "If you write a literal like `0X123` the compiler will suggest `0x123`. This test make sure the compiler doesn't make such suggestions when it shouldn't."